test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -121,14 +121,16 @@ Write-LogPassThru -Message "***** New Run *****"
 Write-LogPassThru -Message "Forcing winrm quickconfig as it is required for remoting tests."
 winrm quickconfig -force
 
-$appVeyorUri = "https://ci.appveyor.com/api"
-$project = Invoke-RestMethod -Method Get -Uri "${appVeyorUri}/projects/PowerShell/powershell-f975h"
+# todo: make this actually work with some
+# new API (this is a placeholder):
+$CIUri = "https://microsoft.com"
```

Comment:
might talk to @adityapatwardhan 	 about the correct fix

Reply:
Let's revert the changes to this file, so we can get the rest of the changes in.

Reply:
Got it

Reply:
Now reverted.